easybuild/easyblocks/a/abaqus.py

```diff
@@ -285,6 +303,9 @@ def sanity_check_step(self):
             custom_paths['dirs'].append('%s-%s' % ('.'.join(verparts[0:2]), verparts[2]))
             custom_commands.append("abaqus information=all")
 
+        if LooseVersion(self.version) >= LooseVersion('2020'):
+            custom_paths['files'].append(os.path.join('cae', 'linux_a64', 'code', 'bin', 'abaqusstd'))
+
```

Comment:
```suggestion
                if self.cfg['with_fe_safe']:
                    custom_paths['files'].append(os.path.join('cae', 'linux_a64', 'code', 'bin', 'fe-safe'))
```

At least I assume that will only appear when installing fe safe. I'm suggesting this in the >= 2020 block as I am not sure where it would be located for older ABAQUS installs.